src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1563,4 +1572,31 @@ public void increaseBoughtCitizenCost()
         boughtCitizenCost = Math.min(1 + (int) Math.ceil(boughtCitizenCost * 1.5), STACKSIZE);
         markDirty();
     }
+
+    /**
+     * Save the time when mercenaries are used, to set a cooldown.
+     */
+    public void usedMercenaries()
+    {
+        mercenaryLastUse = world.getTotalWorldTime();
+        markDirty();
+    }
+
+    /**
+     * Get the last time mercenaries were used.
+     */
+    public long getMercenaryUseTime()
+    {
+        return mercenaryLastUse;
+    }
+
+    /**
+     * Registers an entity related to this colony.
+     */
+    public void registerEntityWithColony(final IColonyRelatedEntity entity)
+    {
+        // Add entity registry logic here
+    }
```

Comment:
is there something missing here?

Reply:
oh thats just unused atm, not sure if we got a use for it yet. Probably should just remove it so we dont have new unused code

Reply:
Agreed